ocs_ci/ocs/node.py

```diff
@@ -55,9 +55,10 @@ def get_typed_nodes(node_type='worker', num_of_nodes=None):
 
     """
     nodes = get_node_objs()
+    from pdb import set_trace; set_trace()
```

Comment:
do we need set_trace here? it will need user input to proceed

Reply:
no :)